docs/src/man/comparisons.md

```diff
@@ -0,0 +1,32 @@
+The table compares the main functions of DataFrames.jl with the R package dplyr (v1) and Stata (>=v8)
+|Operations| DataFrames.jl       | dplyr | Stata|
+|:------------|:------------|:------------|:------------|
+|Reduce multiple values|`combine(df, :x => mean)`|`summarize(df, mean(x))`|`collapse (mean) x =`|
+|Add new columns|`transform(df, :x => mean => :x_mean)`|`mutate(df, x_mean = mean(x))`|`egen x_mean = mean(x)`|
+|Pick columns|`select(df, :x, :y)`|`select(df, x, y)`|`keep x y`|
```

Comment:
I'd also show how to compute new columns using `select`. This is `transmute` in dplyr.